opentelemetry-instrumentation/tests/test_auto_tracing.py

```diff
@@ -104,3 +108,23 @@ def test_trace_init_otlp(self):
             "my-otlp-test-service",
         )
         del environ["OTEL_SERVICE_NAME"]
+
+    def test_trace_init_default_random_ids_generator(self):
+        environ["OTEL_SERVICE_NAME"] = "my-random-ids-generator-test-service"
```

Comment:
Just a [suggestion](https://docs.python.org/3/library/unittest.mock.html#patch-dict): `@patch.dict('os.environ', {'newkey': 'newvalue'})`

Reply:
Sweet thank you! Didn't know about this 😄